src/lumigo_opentelemetry/__init__.py

```diff
@@ -184,7 +210,35 @@ def init() -> Dict[str, Any]:
 
     trace.set_tracer_provider(tracer_provider)
 
-    return {"tracer_provider": tracer_provider}
+    if logging_enabled:
+        _logs.set_logger_provider(logger_provider)
+
+        # Add the handler to the root logger, hence affecting all loggers created by the app from now on
+        logging.getLogger().addHandler(LoggingHandler(logger_provider=logger_provider))
+
+        # Inject the span context into logs
+        LoggingInstrumentor().instrument(set_logging_format=True)
+
+        if logdump_file:
```

Comment:
optional load the logdump_file env var only if logging is enabled?

Reply:
I don't mind, I just followed the same pattern used by the span-dump env-var for consistency. Actually since loading the env-var is cheap, I'm up for pre-checking all of them and relying on variables later on